src/gsy_e/models/matching_engine_matcher/matching_engine_external_matcher.py

```diff
@@ -26,11 +26,14 @@
 from gsy_framework.enums import AvailableMarketTypes
 
 import gsy_e.constants
-from gsy_e.gsy_e_core.exceptions import InvalidBidOfferPairException, MycoValidationException
+from gsy_e.gsy_e_core.exceptions import (InvalidBidOfferPairException,
+                                         MatchingEngineValidationException)
 from gsy_e.gsy_e_core.market_counters import ExternalTickCounter
-from gsy_e.gsy_e_core.redis_connections.area_market import myco_redis_communicator_factory
+from gsy_e.gsy_e_core.redis_connections.area_market import \
```

Comment:
Please do not use backslashes for lines that exceed the maximum character limit. Please follow the approach that you did in line 29, use brackets. 
Thanks a lot!